README.md

```diff
@@ -1,41 +1,41 @@
 
-# Firehose-to-Logdna
+# Firehose-to-LogDNA
 
-This is a Nozzle to drain log from firehose in CloudFoundry to LogDNA.
+This is a Nozzle to send firehose events from CloudFoundry to LogDNA.
 
 Fork From: https://github.com/cloudfoundry-community/firehose-to-syslog
 
 # Usage
 
-Currently the Nozzle only run as an app in Cloud Foundry.
+Currently the Nozzle only runs as an app in Cloud Foundry.
 
-1 - Download the latest release of Firehose-to-Logdna.
+1. Download the latest release of Firehose-to-LogDNA.
 ```
-    $git clone https://github.com/logdna/logdna-cloudfoundry.git
-    $cd firehose-to-syslog
+    git clone https://github.com/logdna/logdna-cloudfoundry.git
+    cd firehose-to-syslog
 ```
 
-2 - Utilize the CF cli to authenticate with your PCF instance.
+2. Utilize the CF cli to authenticate with your PCF instance.
 ```
     cf login -a https://api.[your cf system domain] -u [your id] --skip-ssl-validation
 ```
 
-3 - Push firehose-to-logdna.
+3. Push firehose-to-logdna.
 ```
     cf push firehose-to-logdna --no-start
 ```
 
-4 - Set environment variables with cf cli.
+4. Set environment variables with cf cli.
 ```
     cf set-env firehose-to-logdna API_ENDPOINT https://api.[your cf system domain]
     cf set-env firehose-to-logdna DOPPLER_ENDPOINT wss://doppler.[your cf system domain]:443
-    cf set-env firehose-to-logdna INGESTION_KEY [your ingestion key in LogDNA]
+    cf set-env firehose-to-logdna INGESTION_KEY [your LogDNA ingestion key]
```

Comment:
I'd suggest `LogDNA Ingestion Key` instead of `your LogDNA ingestion key`

Reply:
Can we leave the "your"? It's not a universal key, it's different for each customer and I think "your" makes that clearer.

Reply:
Yup - no harm.